lib/ddtrace/contrib/active_support/notifications/subscription.rb

```diff
@@ -28,18 +28,17 @@ def after_trace(&block)
             @after_trace_callbacks << block if block_given?
           end
 
+          def call(name, start, _, id, payload)
```

Comment:
If `call` and `start`/`finish` both exist, which does ActiveSupport::Notifications use? Does it call both?

Reply:
ActiveSupport 4 and 5 appear to just call `start/finish`, and ActiveSupport 3 calls `call`. I ran into this when testing the ActiveModelSerializers using `contrib-old`

Reply:
Ah good to know.